src/components/drawing/index.js

```diff
@@ -360,8 +360,23 @@ drawing.pointStyle = function(s, trace, gd) {
     });
 };
 
+drawing.waterfallStyle = function(d, trace) {
+    var marker = trace.marker;
+
+    if(d.isSum) {
+        return trace.totals || marker;
+    } else {
+        if(d.rawS > 0) return trace.increasing || marker;
+        if(d.rawS < 0) return trace.decreasing || marker;
```

Comment:
Hmm. This logic hasn't been updated it looks like.

@archmoj did we agree that `marker.color` if set would take precedence over `(increasing|decreasing|totals).marker.color`